### ISSUE TITLE
Thread panicked - "creation time is not available for the filesystem" 

### ISSUE THREAD

**Issue by MostHated** [ogham/exa#832]
Hey there,
    I simply tried to run a custom alias of 'la' (seen below) after changing directories to my mounted QNAP NAS. Some commands work fine, others do not. 

- The version of exa being used (`exa --version`)
```sh
zsh ✗  exa --version 
exa v0.9.0
```
- The command-line arguments you are using
```sh
exa -lhUma --time-style=default --git --color-scale
```
- Your operating system and hardware platform

<details>
<summary>System Details</summary>

```sh
OS: Pop!_OS 20.10 x86_64
Host: X570 AORUS ELITE -CF
Kernel: 5.11.0-7612-generic
Uptime: 2 days, 11 hours, 42 mins
Packages: 3311 (dpkg), 39 (flatpak), 9 (snap)
Shell: zsh 5.8
Resolution: 1920x1080, 2560x1080, 1920x1080
DE: GNOME 3.38.3
WM: Mutter
WM Theme: Pop
Theme: Material-Black-Mango-3.38 [GTK2/3]
Icons: Future-dark-cursors [GTK2/3]
Terminal: alacritty
CPU: AMD Ryzen 9 3950X (32) @ 3.500GHz
GPU: NVIDIA GeForce RTX 2070
Memory: 22445MiB / 64316MiB
```

</details>

<details>
<summary>Full Error</summary>

```sh
zsh >> la                                                                                                                                     mosthated@pop-os
thread '<unnamed>thread 'thread '' panicked at '<unnamed>thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }', ' panicked at 'src/fs/file.rscalled `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }', thread 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }thread ':', src/fs/file.rs<unnamed>', <unnamed>thread '346:src/fs/file.rs' panicked at 'src/fs/file.rs' panicked at '<unnamed>:3469:called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }:called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }' panicked at ':
346', 346', called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "creation time is not available for the filesystem" }9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
:[1]    1876747 abort (core dumped)  exa -lhUma --time-style=default --git --color-scale
```

</details>

---

Command List
```PowerShell
<# Works #> alias l=  'exa -lbF --git' 
<# Works #> alias ls= 'exa'                                                         
<# Works #> alias ld= 'exa -lbF --git --sort mod' 
<# Works #> alias ll= 'exa -lbGF --git' 
<# Works #> alias llm='exa -lbGd --git --sort=modified' 
<# Works #> alias lS= 'exa -1'                                                              
<# Works #> alias lt= 'exa --tree --level=2' 
<# Error #> alias la= 'exa -lhUma --time-style=default --git --color-scale' # -- I tried removing '--time-style' to see if it would make a difference, but no.
<# Error #> alias lx= 'exa -lbhHigUmuSa@ --time-style=long-iso --git --color-scale' 
```

If any additional information is needed, please do let me know.
Thanks,
-MH

**Comment by lespea** [comment]
exa v0.10.0 should have fixed this issue

**Comment by MostHated** [comment]
Ah, ok, nice. I guess it has not made it's way to package managers yet. I just checked with apt and there was no update yet.

**Comment by ariasuni** [comment]
Duplicate of #748, fixed by #616, which means it’s fixed since exa 0.10.0. Thanks for reporting anyway. :)

Please wait for the update in your distribution or [install exa manually](https://the.exa.website/install/linux).
